Disable boolean values for validates_start_with_lowercase and remove deprecation warning (#6077)
realm/SwiftLint: CHANGELOG.md

### Breaking

> * Removed support for boolean values in the `validates_start_with_lowercase` option
>   and its deprecation warning. Use severity levels (`off`, `warning`, or `error`) instead.
>   [kaseken](https://github.com/kaseken)

Reviewer: ```suggestion
* Remove support for boolean values in the `validates_start_with_lowercase` option entirely.
  Use the severity levels `off`, `warning` or `error` instead.  
  [kaseken](https://github.com/kaseken)
```
